### DISCUSSION TITLE
Missing "Project" field in issue buffer

### DISCUSSION THREAD

**Discussion by CptnGreen** [magit/forge#755] [Ideas]
I plan to use kanban in my current collaborative project so i miss an opportunity to assign issue to the board (they call boards "Projects" on github...). Please consider including this little feature.

**Comment by mankoff** [comment]
This would be a great feature. Until it's fully supported by Forge, you could collect the needed info (issue number is on current line or in frame title, depending on view, etc.) and call the `gh` CLI script to do this for you, assuming that is easier to get working than a robust elisp implementation contributed as a PR.

**Comment by tarsius** [comment]
I hesitate to add this feature because it is only the first step toward full support for projects and I don't have the bandwidth to implement that any time soon.

I believe to remember that there already is a package that adds support for projects, though I might be imagining things.

**Comment by arielshaqed** [comment]
I too could use something like this!

I see `project-cards` [in the source for issues](https://github.com/magit/forge/blob/5dcca0e0c0378499b086543be2ec292ddd8beb5b/lisp/forge-issue.el#L51), and also for pull-requests.  And I'm hoping that that was a first step, perhaps?

**Comment by ps1ke** [comment]
Is this feature still way down in your backlog? This is the only feature that is making me still having to open up github.com. At my org we use projects to organize us

**Reply by idlip** [reply]
+1

tbh in github project part is slow to act on.
Wish forge makes some minimal implementation to easily push issues/pr to project board (same as labels)

**Comment by tarsius** [comment]
I plan to move away from forges such as Github. I was planning to move to Codeberg, but am not sure anymore, that is enough. Developments over the last few and especially the last year, forced me to invest in defense, and I am increasingly worried that having a world-writable public todo list puts maintainers under constant stress. I'll keep maintaining Forge; after all most (i.e., currently all) of my packages are still hosted on Github and I also want to keep using it for repositories on Github, which are owned by other people. But new features are becoming increasingly unlikely.

I am least motivated to add features like this, which increase lockin.
